cmd/bootnode/main.go

```diff
@@ -15,7 +15,8 @@
 // along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
 
 // bootnode runs a bootstrap node for the Ethereum Discovery Protocol.
-package main
+// Keep package as bootnode during upstram merge.
+package bootnode
```

Comment:
Why is this enforced?

Reply:
I am trying to import functions from this module, we don't use a different bootnode binary like in case of geth.